api-server/tox.ini

```diff
@@ -12,28 +14,36 @@ setenv =
     CHARM_BRANCH=main
     LOCAL_CHARM_DIR=charm_repo
 
-[testenv:unit]
+[testenv:pack]
 passenv = *
 allowlist_externals =
-    bash
-    tox
     rockcraft
-deps =
-    juju~=2.9.0
-    pytest
-    pytest-operator
-    ops
 commands =
-    # build and pack rock
     rockcraft pack
+
+[testenv:export-to-docker]
+passenv = *
+allowlist_externals =
+    bash
+    skopeo
+    yq
+commands =
+    # pack rock and export to docker
     bash -c 'NAME=$(yq eval .name rockcraft.yaml) && \
              VERSION=$(yq eval .version rockcraft.yaml) && \
-             ARCH=$(yq eval ".platforms | keys" rockcraft.yaml | awk -F " " '\''{ print $2 }'\'') && \
-             ROCK="$\{NAME\}_$\{VERSION\}_$\{ARCH\}" && \
-             sudo skopeo --insecure-policy copy oci-archive:$ROCK.rock docker-daemon:$ROCK:$VERSION && \
-             docker save $ROCK > $ROCK.tar'
-    # run rock tests
-    pytest -v --tb native --show-capture=all --log-cli-level=INFO {posargs} {toxinidir}/tests
+             ARCH=$(yq eval ".platforms | keys | .[0]" rockcraft.yaml) && \
+             ROCK="$\{NAME\}_$\{VERSION\}_$\{ARCH\}.rock" && \
+             DOCKER_IMAGE=$NAME:$VERSION && \\
+             echo "Exporting $ROCK to docker as $DOCKER_IMAGE" && \
+             skopeo --insecure-policy copy oci-archive:$ROCK docker-daemon:$DOCKER_IMAGE'
+
+[testenv:sanity]
+passenv = *
+allowlist_externals = 
+    echo
+commands =
+    # TODO: Implement sanity tests
+    echo "WARNING: This is a placeholder test - no test is implemented here."
```

Comment:
Completely makes sense for this task.  Do we have an issue somewhere reminding us to refactor our rock sanity tests so we are consistent across different repos?  That way we dont forget in future

Reply:
Created this issue https://github.com/canonical/pipelines-rocks/issues/59 and linked it to https://github.com/canonical/bundle-kubeflow/issues/783 also. Not sure if and where should we have an issue to track effort across all repos.